tests/validate/bandwidth_test/src/host.cpp

```diff
@@ -151,102 +159,171 @@ int main(int argc, char** argv) {
 
     double max_throughput = 0;
     int reps = stoi(iter_cnt);
+    if (num_kernel_ddr) {
+        for (uint32_t i = 4 * 1024; i <= 16 * 1024 * 1024; i *= 2) {
```

Comment:
Please add a comment on what you are doing here.  For example:
Starting at 4K and going up to 16M with increments of power of 2 ....